TestPlaybooks/playbook-Jask_Test.yml

```diff
@@ -520,13 +536,42 @@ tasks:
         }
       }
     note: false
+    timertriggers: []
+  "21":
+    id: "21"
+    taskid: 106d1f90-6828-4d80-8755-44f1a2a251d0
+    type: regular
+    task:
+      id: 106d1f90-6828-4d80-8755-44f1a2a251d0
+      version: -1
+      name: Test Fetch Incidents
+      scriptName: FetchFromInstance
+      type: regular
+      iscommand: false
+      brand: ""
+    scriptarguments:
+      add_to_context: {}
```

Comment:
Would be nice if we verify the context at least on some level to make sure we actually pull some data